### ISSUE TITLE
Exclusion list

### ISSUE THREAD

**Issue by bbmaster123** [Maplespe/DWMBlurGlass#78]
Very cool app! This makes windows look like how I thought it should have looked 10 years ago! :)

I have a few apps that I want to keep as Mica/Tabbed, can you consider adding an exclusion list eventually? No hurry, thanks again!

**Comment by Oronhyatekha** [comment]
Yeah I also need it for an App like Crosshair X. Because when it's pinned in Game Bar, the whole screen is blurry.

**Comment by Maplespe** [comment]
@Oronhyatekha The problem of unusual blurring of other applications has been solved in the new version.

**Comment by harphere** [comment]
Sorry, but where do I find the exclusion option? I'm running 2.3.2 Beta 3

Awesome tool by the way!

**Comment by RaXorX** [comment]
@Maplespe How exactly is it solved? I noticed a few old apps I use also turned to full on Blur. Like literal window turned blurry. A little curious why we don't have an inclusion/exclusion list unless it exists and I can't see it.
